webpack.config.js

```diff
@@ -4,6 +4,8 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const nodeExternals = require("webpack-node-externals");
 const CopyPlugin = require("copy-webpack-plugin");
 const { ServiceWorkerPlugin } = require("service-worker-webpack");
+const { bundledSyntaxHighlighters } = require("./src/shared/build-config");
+const { lazySyntaxHighlighters } = require("./src/shared/build-config");
```

Comment:
If I'm not mistaken, this can be shortened to:

```suggestion
const { bundledSyntaxHighlighters, lazySyntaxHighlighters } = require("./src/shared/build-config");
```